autoreject/utils.py

```diff
@@ -356,22 +356,20 @@ def _interpolate_bads_meg_fast(inst, picks, mode='accurate', verbose=None):
     else:
         picked_info = inst.info.copy()
 
-    def get_picks_bad_good(info):
-        picks_meg = pick_types(info, meg=True, eeg=False, exclude=[],
-                               stim=False)
```

Comment:
@dengemann this was the buggy part. If the user passes `picks` which has only `grad` channels, then the `picks_meg` here is not accurate because it does not know that it should only pick `grad` channels

Reply:
Outch. I do not dare to query git blame ...